MapboxDirections/MBDirections.swift

```diff
@@ -173,6 +173,9 @@ open class Directions: NSObject {
                     route.accessToken = self.accessToken
                     route.apiEndpoint = self.apiEndpoint
                     route.routeIdentifier = json["uuid"] as? String
+                    if let vLocale = json["voiceLocale"] as? String {
+                        route.spokenLocale = Locale(identifier: vLocale)
```

Comment:
What do you think about also copying this locale into each SpokenInstruction, so that the navigation SDK’s voice controller classes only need to work with individual SpokenInstruction objects?

Reply:
Yeah that works